src/bigint.rs

```diff
@@ -66,6 +66,24 @@ impl Mul<Sign> for Sign {
     }
 }
 
+impl<'a, T: Integer> Pow<&'a T> for Sign {
```

Comment:
I'm still hesitating whether we really want a public `impl` for this.  It's kind of silly to have a bunch of traits implemented for `Sign`.  (An alternative is to just have a private `fn` to get the new sign.)

Reply:
Private fn :heavy_check_mark:

Reply:
I'd have expected the entire `Sign` trait private by the way, but there's probably some good reason for it.

Reply:
It's an `enum Sign`, not a trait, and it's a public argument for some of the constructors.  We have to indicate the sign *somehow*, since we're not using 2's complement, so an `enum` is relatively clean.  It usually stays out of the way. :)